theme.json

```diff
@@ -288,90 +295,90 @@
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata60pt-Black.woff2"
+								"file:./assets/fonts/literata/Literata60pt-SemiBold.woff2"
 							],
-							"fontWeight": "900",
+							"fontWeight": "600",
 							"fontStyle": "normal",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata60pt-BlackItalic.woff2"
+								"file:./assets/fonts/literata/Literata60pt-BoldItalic.woff2"
 							],
-							"fontWeight": "900",
+							"fontWeight": "700",
 							"fontStyle": "italic",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata60pt-BoldItalic.woff2"
+								"file:./assets/fonts/literata/Literata-Bold.woff2"
 							],
 							"fontWeight": "700",
-							"fontStyle": "italic",
+							"fontStyle": "normal",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata60pt-ExtraBold.woff2"
+								"file:./assets/fonts/literata/Literata-BoldItalic.woff2"
```

Comment:
This one looks like a duplicate.

Reply:
It might be because of the name are different. I'll check them manually again and fix the duplicates. 

Thank you for sharing the ideas about checking the duplicates.